src/source/PeerConnection/PeerConnection.h

```diff
@@ -21,9 +21,9 @@ extern "C" {
 // A non-comprehensive list of valid JSON characters
 #define VALID_CHAR_SET_FOR_JSON         "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/"
 
-#define ICE_CANDIDATE_JSON_TEMPLATE     (PCHAR) "{\"candidate\":\"candidate:%s\",\"sdpMid\":\"0\",\"sdpMLineIndex\":0}"
-
-#define MAX_ICE_CANDIDATE_JSON_LEN      (MAX_SDP_ATTRIBUTE_VALUE_LENGTH + STRLEN(ICE_CANDIDATE_JSON_TEMPLATE) + 1)
+#define ICE_CANDIDATE_JSON_TEMPLATE     (PCHAR) "{\"candidate\":\"candidate:%s\",\"sdpMid\":\"0\",\"sdpMLineIndex\":0}" // SIZE 59
+ 
+#define MAX_ICE_CANDIDATE_JSON_LEN      (MAX_SDP_ATTRIBUTE_VALUE_LENGTH + 59 + 1) // (MAX_SDP_ATTRIBUTE_VALUE_LENGTH + STRLEN(ICE_CANDIDATE_JSON_TEMPLATE) + 1)
```

Comment:
Are you worried about the cost of `STRLEN` at runtime? Would it be possible to do a SIZEOF instead, just so we don't need to hardcode the 59.

Reply:
Not at all but Windows is not happy using STRLEN because of is a not const expression

Reply:
https://github.com/wolfviking0/amazon-kinesis-video-streams-webrtc-sdk-c/blob/191754a7b0a15edeb5dd242523ebe9e3f73bd3d8/src/source/PeerConnection/PeerConnection.c#L317
This line failed on Windows if you don't remove the strlen

Reply:
Usage of ARRAY_SIZE() is a compile-time directive which is the right way to go. NOTE: ARRAY_SIZE will return the null terminator so one more char to account for